ui/desktop/app/routes/scopes/scope/projects.js

```diff
@@ -76,41 +70,45 @@ export default class ScopesScopeProjectsRoute extends Route {
   }
 
   willDestroy() {
```

Comment:
I'm wondering if we should be using `activate` and `deactivate` like in the OIDC route?

Reply:
yeah thats a good point on using `deactivate` instead since its called when the route is completed exited and aligns with this use case. on the point of using activate, i think keeping the logic in the model hook works best since `activate` hook is not called on refreshes.